src/SldStyleParser.ts

```diff
@@ -695,18 +701,28 @@ export class SldStyleParser implements StyleParser<string> {
     if (!isNil(haloColor)) {
       textSymbolizer.haloColor = haloColor;
     }
-    const displacement = get(sldSymbolizer, 'LabelPlacement.PointPlacement.Displacement');
-    if (!isNil(displacement)) {
-      const x = get(displacement, 'DisplacementX.#text');
-      const y = get(displacement, 'DisplacementY.#text');
-      textSymbolizer.offset = [
-        Number.isFinite(x) ? numberExpression(x) : 0,
-        Number.isFinite(y) ? numberExpression(y) : 0,
-      ];
-    }
-    const rotation = get(sldSymbolizer, 'LabelPlacement.PointPlacement.Rotation.#text');
-    if (!isNil(rotation)) {
-      textSymbolizer.rotate = numberExpression(rotation);
+    const placement = get(sldSymbolizer, 'LabelPlacement');
+    if (!isNil(placement)) {
+      const pointPlacement = get(placement, 'PointPlacement');
+      const linePlacement = get(placement, 'LinePlacement');
+      if (!isNil(pointPlacement)) {
+        textSymbolizer.placement = 'point';
+        const displacement = get(placement, 'PointPlacement.Displacement');
+        if (!isNil(displacement)) {
+          const x = get(displacement, 'DisplacementX.#text');
+          const y = get(displacement, 'DisplacementY.#text');
+          textSymbolizer.offset = [
+            Number.isFinite(x) ? numberExpression(x) : 0,
+            Number.isFinite(y) ? numberExpression(y) : 0,
+          ];
+        }
+        const rotation = get(placement, 'PointPlacement.Rotation.#text');
```

Comment:
Shouldn't this be:
```suggestion
        const rotation = get(pointPlacement, 'Rotation.#text');
```

Reply:
solved in https://github.com/geostyler/geostyler-sld-parser/pull/916/commits/9c455a64287da0dbd7ebc628705d7b575f52236b